octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -57,6 +128,201 @@ async def matrix_callback(self,
         # Do not forget to check if evaluator_name is self.name
         pass
 
+    def _are_every_evaluation_valid(self,
+                                    matrix_id,
+                                    evaluator_name,
+                                    evaluator_type,
+                                    exchange_name,
+                                    cryptocurrency,
+                                    symbol,
+                                    time_frame
+                                    ):
+        available_time_frames = self.get_available_time_frames(matrix_id, exchange_name, evaluator_type,
```

Comment:
I don't understand why you don't just use `self.strategy_time_frames` here

Reply:
A strategy might require a timeframe that is not available in an exchange, therefore we have to check within exchange available time frames

Reply:
Oh yes ok I understand